spaceship/lib/spaceship/test_flight/build_trains.rb

```diff
@@ -19,6 +19,13 @@ def self.all(app_id: nil, platform: nil, retry_count: 3)
       included = []
       cursor = nil
 
+      prerelease_platform = case platform
+                            when "appletvos"
+                              "TV_OS"
+                            else
+                              "IOS"
+                            end
```

Comment:
If someone has watchos or macos apps it would be helpful to know what codes are used by the api